universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref.csl

```diff
@@ -0,0 +1,935 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" et-al-min="3" et-al-use-first="1" initialize="false" demote-non-dropping-particle="never" default-locale="fr-CA" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Université du Québec à Montréal - APA - prénoms - Études féministes - IREF (Français - Canada)</title>
+    <title-short>UQAM prénoms</title-short>
+    <id>http://www.zotero.org/styles/universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref</id>
+    <link href="http://www.zotero.org/styles/universite-du-quebec-a-montreal-prenoms-etudes-feministes-iref" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
```

Comment:
You've mentioned that this is an almost exact copy of another style. Please add another template node.

Reply:
Yes, go with @bwiernik's suggestions. That's clearer.

Reply:
I try to delete line 8 in the Edit mode, but the "Commit changes" button is not active. I am not sure how I can save my commit.

Reply:
Are you working online or locally?

Reply:
I work online (as I usually do).

Reply:
It works today on my computer at work. It was not working at home.